content/300-guides/200-deployment/101-traditional-deployments/200-deploying-to-heroku.mdx

```diff
@@ -17,7 +17,7 @@ The application has the following components:
 - **Backend**: Node.js REST API built with Express.js with resource endpoints that use Prisma Client to handle database operations against a PostgreSQL database (e.g., hosted on Heroku).
 - **Frontend**: Static HTML page to interact with the API.
 
-![architecture diagram](../images/heroku-architecture.png)
+![architecture diagram](./images/heroku-architecture.png)
```

Comment:
Would probably make sense to move the images to the subfolder as well to make them easier to attribute and manage. Would keep the image tags in the files unchanged as well.